prompt.go

```diff
@@ -109,49 +110,39 @@ type PromptTemplates struct {
 // Run will keep the prompt alive until it has been canceled from the command prompt or it has received a valid
 // value. It will return the value and an error if any occurred during the prompt's execution.
 func (p *Prompt) Run() (string, error) {
-	c := &readline.Config{}
```

Comment:
apologies for the added churn here. I can put it back if that's desirable-- I found the logic to arrive at a config distracting, especially when many of the values passed into readline ultimately don't require any checking/modification.